opennms-provision/opennms-detector-simple/src/main/java/org/opennms/netmgt/provision/detector/snmp/Win32ServiceDetector.java

```diff
@@ -63,15 +65,11 @@ public String getWin32ServiceName() {
     public void setWin32ServiceName(String serviceName) {
         m_win32SvcName = serviceName;
         LOG.debug("setWin32ServiceName: setting service name to {}", serviceName);
-        int snLength = serviceName.length();
-        
-        StringBuffer serviceOidBuf = new StringBuffer(SV_SVC_OPERATING_STATE_OID);
-        serviceOidBuf.append(".").append(Integer.toString(snLength));
-        for (byte thisByte : serviceName.getBytes()) {
```

Comment:
I added `extend dëadbeef /usr/bin/env echo dëad beef cafe` to /etc/snmp/snmpd.conf on my centos 7 linux box (net-snmp 5.7.3), and the table returns this:

```
# snmpbulkwalk -Oun -c public -v 2c localhost enterprises.8072.1.3
.1.3.6.1.4.1.8072.1.3.2.1.0 = INTEGER: 1
.1.3.6.1.4.1.8072.1.3.2.2.1.2.9.100.195.171.97.100.98.101.101.102 = STRING: /usr/bin/env
.1.3.6.1.4.1.8072.1.3.2.2.1.3.9.100.195.171.97.100.98.101.101.102 = STRING: echo d..ad beef cafe
.1.3.6.1.4.1.8072.1.3.2.2.1.4.9.100.195.171.97.100.98.101.101.102 = STRING: 
.1.3.6.1.4.1.8072.1.3.2.2.1.5.9.100.195.171.97.100.98.101.101.102 = INTEGER: 5
.1.3.6.1.4.1.8072.1.3.2.2.1.6.9.100.195.171.97.100.98.101.101.102 = INTEGER: exec(1)
.1.3.6.1.4.1.8072.1.3.2.2.1.7.9.100.195.171.97.100.98.101.101.102 = INTEGER: run-on-read(1)
.1.3.6.1.4.1.8072.1.3.2.2.1.20.9.100.195.171.97.100.98.101.101.102 = INTEGER: permanent(4)
.1.3.6.1.4.1.8072.1.3.2.2.1.21.9.100.195.171.97.100.98.101.101.102 = INTEGER: active(1)
.1.3.6.1.4.1.8072.1.3.2.3.1.1.9.100.195.171.97.100.98.101.101.102 = STRING: d..ad beef cafe
.1.3.6.1.4.1.8072.1.3.2.3.1.2.9.100.195.171.97.100.98.101.101.102 = STRING: d..ad beef cafe
.1.3.6.1.4.1.8072.1.3.2.3.1.3.9.100.195.171.97.100.98.101.101.102 = INTEGER: 1
.1.3.6.1.4.1.8072.1.3.2.3.1.4.9.100.195.171.97.100.98.101.101.102 = INTEGER: 0
.1.3.6.1.4.1.8072.1.3.2.4.1.2.9.100.4294967235.4294967211.97.100.98.101.101.102.1 = STRING: d..ad beef cafe
```
ë got converted to 195.171 and 4294967235.4294967211 for two of the index values. but when it displayed it, it just showed it as two consecutive periods.

Win32ServiceMonitor also duplicates the code, so changes should also be made to it.